docs/schema/reference.md

```diff
@@ -455,3 +455,16 @@ The publisher block is used in release and record packages to identify the sourc
 ```{jsonschema} ../../build/current_lang/release-package-schema.json
 :include: publisher
 ```
+
+### Link
+
+The `Link` object is used in the `links` field:
+
+```{field-description} ../../build/current_lang/release-schema.json /properties/links
+```
+
+```{jsonschema} ../../build/current_lang/release-schema.json
+:pointer: /definitions/Link
+```
+
+Links are used to reference related resources that describe the *same* release or contracting process. Links should not be used to reference related contracting processes. To reference related processes, see [RelatedProcess](#relatedprocess).
```

Comment:
Actually, the description in the schema is already clear: https://github.com/open-contracting/standard/blob/1.2-dev/schema/release-schema.json#L122

Instead of this paragraph, we should just cite the schema description (as in this PR using `field-description`).

To make the point about using `relatedProcesses`, we should instead add to the schema description, e.g.:

> For references to related processes, see the `relatedProcesses` array.